packages/closer/pages/subscriptions/citizen/apply.tsx

```
@@ -0,0 +1,257 @@
+  const [copied, setCopied] = useState(false);
+
+  const monthlyCreditsSelected = Math.min(
+    parseFloat(monthlyCredits as string) || selectedPlan?.monthlyCredits || 0,
+    MAX_CREDITS_PER_MONTH,
+  );
+  const defaultConfig = useConfig();
+  const PLATFORM_NAME =
+    generalConfig?.platformName || defaultConfig.platformName;
+
+  useEffect(() => {
+    if (user?.subscription && user.subscription.priceId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)